### PR TITLE
Team B Unitree Go1 Pull Request

### PR DESCRIPTION
**Pull request by TingWeiWong**

### CODE REVIEW

**Comment by TingWeiWong** [comment]
Added docker security wiki page

File: assets/images/docker_socket.png
Comment (smbryan) [inline assets/images/docker_socket.png:1]: These asset files shouldn't be in the root-level assets, but rather `wiki/tools/assets` or `wiki/common-platforms/assets`.

File: wiki/common-platforms/unitree-go1.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+# You should set the date the article was last updated like this:
+date: 2023-05-03 # YYYY-MM-DD
+# This will be displayed at the bottom of the article
+# You should set the article's title:
+title: Unitree Go1 Edu 
+# The 'title' is automatically displayed at the top of the page
+# and used in other parts of the site.
+---
+
+(Note: This section contains the latest information as of May 2023)
```
Comment (smbryan) [inline wiki/common-platforms/unitree-go1.md:13]: I'd recommend moving this to the bottom of the introduction.

File: wiki/common-platforms/unitree-go1.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+This is an article that provides an overview of the Unitree Go1 Edu robot, including its features and capabilities. Unitree Robotics is a leading Chinese manufacturer that specializes in developing, producing, and selling high-performance quadruped robots. One of the company's primary advantages is that they offer quadruped platforms at a significantly lower cost compared to competitors like Boston Dynamics. In addition, they have announced plans to release experimental humanoid platforms in the near future.
+
+There are three versions of the Unitree Go1: Air, Pro, and Edu. The Edu model is designed for educational purposes and provides developers with access to the platform. In this article, we will focus on the capabilities of the Go1 Edu, which is a popular choice for students and researchers due to its affordability and ease of use.
+
+## Form Factor
+![Form_Factor](../../assets/images/form_factor.png)
```
Comment (smbryan) [inline wiki/common-platforms/unitree-go1.md:20]: Please move the assets to the parent assets directory. Then, this line should read `Form_Factor`

**Review by smbryan — CHANGES_REQUESTED** [review]
Excellent start. A few more items are required before we can pull this in.

**Review by nevalsar — CHANGES_REQUESTED** [review]
Please see and resolve existing review comments.

**Comment by TingWeiWong** [comment]
Fixed image path issues.

File: wiki/common-platforms/assets/unitree_side.png
Comment (nevalsar) [inline wiki/common-platforms/assets/unitree_side.png:1]: Image is corrupted and not viewable - please replace.

**Review by nevalsar — CHANGES_REQUESTED** [review]
Thank you for fixing the earlier issues! 
Please fix the one image that looks corrupted and we can merge this in.

File: wiki/common-platforms/assets/unitree_side.png
Comment (TingWeiWong) [reply wiki/common-platforms/assets/unitree_side.png:1]: Hi, the unitree-side.png on my side is also not displaying. Unfortunately I couldn't find the original image, I'm removing it from the markdown.

**Review by nevalsar — APPROVED** [review]
Looks good! Merging.

**Review by nevalsar — APPROVED** [review]

**Pull request merged**